runtime-testsuite/test/org/antlr/v4/test/runtime/go/TestParserExec.java

```diff
@@ -611,7 +611,7 @@ public void testPredicatedIfIfElse() throws Exception {
 		grammarBuilder.append("grammar T;\n");
 		grammarBuilder.append("s : stmt EOF ;\n");
 		grammarBuilder.append("stmt : ifStmt | ID;\n");
-		grammarBuilder.append("ifStmt : 'if' ID stmt ('else' stmt | { p.GetTokenStream().LA(1)!=TParserELSE }?);\n");
+		grammarBuilder.append("ifStmt : 'if' ID stmt ('else' stmt | { p.GetTokenStream().LA(1)!=TParser.ELSE }?);\n");
```

Comment:
I'm having trouble following. Are you saying I made these changes in a target/generated file instead of a template file? I can't recall the exact context of these changes, other than that they didn't compile because they didn't match the generated code. `TParserELSE`, for example, is generated, not `TParser.ELSE` (note the period). Go doesn't have type/class properties/fields, so the closest you can get is `mypackage.Myvar` or `mypackage.Myconst`. There were similar issues with functions being called that were actually methods, and vice versa.